src/app/view/components/_common/action-button/action-button.component.html

```diff
@@ -1,7 +1,9 @@
 <p-splitButton 
-    icon="fa-plus" 
+    icon="fa-plus"
     class="action-button" 
     (onClick)="command()" 
     [model]="model" 
-    [class.multiple-options]="model && model.length">
-</p-splitButton>
\ No newline at end of file
+    [class.multiple-options]="model && model.length"
+    [class.toolbar-add-contentlet__head-button]="flat">
+</p-splitButton>
+<span *ngIf="label" class="showMore">{{ label }}</span>
```

Comment:
This class name it's not BEM

Reply:
Also I think showMore is not a good name, maybe something like showLabel is better.